apps/studio/components/interfaces/Storage/Storage.constants.ts

```diff
@@ -1,3 +1,6 @@
+import { DOCS_URL } from 'lib/constants'
+
+// Original storage constants
```

Comment:
Soon-to-be outdated storage constants like `STORAGE_SORT_BY_ORDER` will be removed in DEPR-95